### PR TITLE
Enable full row selection in tables

### PR DESCRIPTION
**Pull request by silvercluster**
Update SWT table to support full row selection.

### CODE REVIEW

**Comment by silvercluster** [comment]
Updated the table style to full row selection where I assumed appropriate.

File: src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewStatisticsContingencyTable.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ protected Control createControl(Composite parent) {
         config.setColumnHeaderLayout(CTConfiguration.COLUMN_HEADER_LAYOUT_FILL_EQUAL);
         config.setRowHeaderLayout(CTConfiguration.ROW_HEADER_LAYOUT_DEFAULT);
 
-        this.table = new ComponentTable(parent, SWT.NONE, config);
+        this.table = new ComponentTable(parent, SWT.FULL_SELECTION, config);
```
Comment (prasser) [inline src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewStatisticsContingencyTable.java]: Please not here. This is not a standard SWT table.

File: src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewStatisticsDistributionTable.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ protected Control createControl(Composite parent) {
         config.setColumnHeaderLayout(CTConfiguration.COLUMN_HEADER_LAYOUT_FILL_EQUAL);
         config.setRowHeaderLayout(CTConfiguration.ROW_HEADER_LAYOUT_DEFAULT);
 
-        this.table = new ComponentTable(parent, SWT.NONE, config);
+        this.table = new ComponentTable(parent, SWT.FULL_SELECTION, config);
```
Comment (prasser) [inline src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewStatisticsDistributionTable.java]: Please not here. This is not a standard SWT table.

**Comment by silvercluster** [comment]
Removed full row selection from non-standard SWT tables.

**Pull request merged**